src/styles/card.css

```diff
@@ -0,0 +1,11 @@
+.card-container {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--white);
+    position: absolute;
```

Comment:
You should definitely not need position absolute for this. This will cause the container ignore all position and will be in issue when you use it in the Page components.

Reply:
I thought I needed position absolute and relative to get two things to overlap?
<img width="265" alt="image" src="https://user-images.githubusercontent.com/15824424/235458285-1b3663a6-d473-4ebc-9357-e4e3e25b09fb.png">

Reply:
You should be able to use negative margins to achieve this effect. Add margin-top: -100px to the container component.

Think about when your application goes into Desktop mode, you will have issues with position relative/absolute. And not EVERY card wants to have position absolute, just in this one use case.